nab/detectors/numenta/modelParams/model_params.json

```diff
@@ -61,7 +61,7 @@
             "seed": 1956,
             "spVerbosity": 0,
             "spatialImp": "cpp",
-            "synPermActiveInc": 0.002,
+            "synPermActiveInc": 0.003,
```

Comment:
@breznak These changes made a meaningful difference to the NAB scores - I believe we should all switch to these parameters for anomaly detection.  There is an API in NuPIC for obtaining the best anomaly detection parameters:

nupic.frameworks.opf.common_models.cluster_params.getScalarMetricWithTimeOfDayAnomalyParams() 

That still needs to be updated with these latest parameters from NAB, see:

https://github.com/numenta/NAB/issues/231

I don't know how good the hotgym parameters are. It should probably use the above API!!